packages/esm-sms-app/src/routes.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://json.openmrs.org/routes.schema.json",
+  "backendDependencies": {},
+  "extensions": [
+    {
+      "name": "sms-providers-settings-admin-card-link",
+      "component": "smsAdminCardLink",
+      "slot": "system-admin-page-card-link-slot",
```

Comment:
Does this slot exist?

Reply:
It's for the `System administration card links`

Reply:
That's not really a useful answer to my question, but I searched around GitHub and found it. https://github.com/openmrs/openmrs-esm-admin-tools/blob/8314a4806fd61353427a4d18f62b8341da7c98fd/packages/esm-system-admin-app/src/dashboard/index.component.tsx#L15